driver/main_args.ml

```diff
@@ -724,6 +724,19 @@ let mk_no_strict_formats f =
   \      and instead fix invalid formats.)"
 ;;
 
+let mk_expand_responsefile f =
```

Comment:
Please respect the format: these functions have the same name as the options they describe, with dashes replaced by underscores, so this should be `mk_args` and the next one should be `mk_args0`.